README.md

```diff
@@ -8,15 +8,51 @@ of Minecraft.
 
 ## Requirements
 
-### For Running From Source
+### Running from Source
+In order to run the Amulet Editor from source, you will need to install the following packages and follow the steps to install below:
 - Python 3.7.0+
 - Numpy
 - pyglet
-- NBT
+- [Amulet_NBT](https://github.com/Amulet-Team/Amulet-NBT)
+
+The steps below will install everything needed for Amulet (Instructions for installing Python 3.7 provided when available)
+
+__Note: Make sure to install the `venv` module if it does not come pre-installed with python__
 
-Shortcut: `pip install -r requirements.txt`
+#### Windows
```

Comment:
Maybe we can not separate the instructions, but only separate the steps where there are differences so it'll be shorter

Reply:
I think that's a good idea, how should we go about doing that?